src/misc/ClientDetector.ts

```diff
@@ -382,6 +382,10 @@ export class ClientDetector {
 			indexedDbSupported: this.indexedDb(),
 		}
 	}
+
+	compressionStreamSupported(): boolean {
+		return typeof CompressionStream !== "undefined"
```

Comment:
We cannot use compression stream, it was only introduced in iOS 16.4 and we have to support 15 for now.

This check is only API check, not a language check, this can go into the library itself, it miiiight be okay to tell old iOS that you can't download invoices there I think.